src/EPROMStore.cpp

```diff
@@ -156,161 +158,168 @@ void EEPROMStore::displayContents()
-  uint16_t value = (uint16_t)valLo + (uint16_t)valHi * 256;
-  LOGV4(DEBUG_EEPROM, F("EEPROM: Read16 %x (%d) from %d"), value, value, location);
-  return value;
+    uint8_t valLo  = read(location);
+    uint8_t valHi  = read(location + 1);
+    uint16_t value = (uint16_t) valLo + (uint16_t) valHi * 256;
+    LOGV4(DEBUG_EEPROM, F("EEPROM: Read16 %x (%d) from %d"), value, value, location);
+    return value;
 }
 
 // Helper to update the given location with the given 32-bit value
 void EEPROMStore::updateInt32(EEPROMStore::ItemAddress location, int32_t value)
 {
-  LOGV4(DEBUG_EEPROM, F("EEPROM: Writing32 %x (%l) to %d"), value, value, location);
-  update(location, value & 0x00FF);
-  update(location + 1, (value >> 8) & 0x00FF);
-  update(location + 2, (value >> 16) & 0x00FF);
-  update(location + 3, (value >> 24) & 0x00FF);
+    LOGV4(DEBUG_EEPROM, F("EEPROM: Writing32 %x (%l) to %d"), value, value, location);
+    update(location, value & 0x00FF);
+    update(location + 1, (value >> 8) & 0x00FF);
+    update(location + 2, (value >> 16) & 0x00FF);
+    update(location + 3, (value >> 24) & 0x00FF);
 }
 
 // Helper to read a 32-bit value from the given location
 int32_t EEPROMStore::readInt32(EEPROMStore::ItemAddress location)
 {
-  uint8_t val1 = read(location);
-  uint8_t val2 = read(location + 1);
-  uint8_t val3 = read(location + 2);
-  uint8_t val4 = read(location + 3);
-  uint32_t uValue = (uint32_t)val1 + (uint32_t)val2 * 256 + (uint32_t)val3 * 256 * 256 + (uint32_t)val4 * 256 * 256 * 256;
-  int32_t value = static_cast<int32_t>(uValue);
-  LOGV4(DEBUG_EEPROM, F("EEPROM: Read32 %x (%l) from %d"), value, value, location);
-  return value;
+    uint8_t val1 = read(location);
+    uint8_t val2 = read(location + 1);
+    uint8_t val3 = read(location + 2);
+    uint8_t val4 = read(location + 3);
+    uint32_t uValue
+        = (uint32_t) val1 + (uint32_t) val2 * 256 + (uint32_t) val3 * 256 * 256 + (uint32_t) val4 * 256 * 256 * 256;
+    int32_t value = static_cast<int32_t>(uValue);
+    LOGV4(DEBUG_EEPROM, F("EEPROM: Read32 %x (%l) from %d"), value, value, location);
+    return value;
 }
 
 // Check if the specified item is present in the core set.
 // Returns: true - if marker is present and item flag is set
 //          false - otherwise
 bool EEPROMStore::isPresent(ItemFlag item)
 {
-  uint16_t marker = readUint16(MAGIC_MARKER_AND_FLAGS_ADDR);
+    uint16_t marker = readUint16(MAGIC_MARKER_AND_FLAGS_ADDR);
 
-  unsigned check = (MAGIC_MARKER_MASK | item);
-  unsigned result=(MAGIC_MARKER_VALUE | item);
-  bool res = ((marker & check) == result);
-  LOGV6(DEBUG_EEPROM, F("EEPROM: IsDataPresent (%x). Checking (%x & %x) == %x => %d"), item, marker, check, result, res);
+    unsigned check  = (MAGIC_MARKER_MASK | item);
+    unsigned result = (MAGIC_MARKER_VALUE | item);
+    bool res        = ((marker & check) == result);
+    LOGV6(DEBUG_EEPROM,
+          F("EEPROM: IsDataPresent (%x). Checking (%x & %x) == %x => %d"),
```

Comment:
Urgh. What makes this go over multiple lines? Can we make it stay on one line?

Reply:
It splits the lines up because of the character limit (120), lets check back on this when we bump that up